script.js

```diff
@@ -721,7 +721,8 @@ function loadDataFromUrlIfPresent() {
 
     $("input[value$='" + paramType + "']").trigger('click');
     $("#dataInput_" + paramType).html(url.searchParams.get("data"));
-    $("#val_" + paramType).trigger('click');
+    $("#generateViewButtonId").trigger('click');
+    return true;
```

Comment:
This is the actual bug fix. We never returned any "truthy" value from this method, so we always fell back to using the cached data